Library/Homebrew/keg.rb

```diff
@@ -529,6 +529,8 @@ def binary_executable_or_library_files
     elf_files
   end
 
+  def codesign_patched_binary(_binary_file); end
```

Comment:
This reintroduces the codesigning regression: https://github.com/Homebrew/brew/pull/12960